ogcserver/WMS.py

```diff
@@ -47,10 +48,12 @@ def __init__(self, configpath=None):
         self.styles = {}
         self.aggregatestyles = {}
         self.map_attributes = {}
+        self.map_scale = 1
```

Comment:
This makes sure the `map_scale` is set by default, so the guard in common.py should not be needed, unless someone decided to write their own BaseWMSFactory.